functiondefextractor/condition_checker.py

```diff
@@ -67,11 +68,15 @@ def get_pivot_table_result(data, test_assert, splitter, file_path):
     data_table = data_table.to_frame()
     data_table = data_table.reset_index()
     data_table = data_table.rename({'index': 'Different %s patterns ' % test_assert}, axis='columns')
+    if data_table.iat[0, 0] == '':  # pragma: no mutate
+        data_table = data_table.drop([data_table.index[0]])  # pragma: no mutate
```

Comment:
What significance with this check here ?

Reply:
Thanks for the suggestion. It's is nor required. Removed